### PR TITLE
For review: Release notes 6.6.1

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -4,6 +4,217 @@ Couchbase Server 6.6 adds features and enhancements to improve developer product
...
+| *Summary*: Improved array indexing performance by optimizing the ComputeArrayEntriesWithCount method.
+
+| https://issues.couchbase.com/browse/MB-41717[MB-41717^]
+| *Summary*: When bloomDelta is added after recovery when page is found without a bloom filter, the stat NumRecordAllocs is over counted. However, NumRecordAllocs is only supposed to track the insert/delete deltas. This has been fixed.
+
+| https://issues.couchbase.com/browse/MB-41155[MB-41155^]
+| *Summary*: Fixed an issue with memory optimized indexes where indefinite disk snapshotting led to increasing disk usage.
+
+| https://issues.couchbase.com/browse/MB-41091[MB-41091^]
+| *Summary*: The debugger link has been updated, from `chrome-devtools://` to `devtools://`,  to adapt to different Chrome versions.
```
Comment (mihirkamdarqe) [inline modules/release-notes/pages/relnotes.adoc]: This issue should be listed in the Eventing section

**Review by mihirkamdarqe — COMMENTED** [review]
MB-41091 listed in Indexer section should be moved to the Eventing section

File: modules/release-notes/pages/relnotes.adoc (hunk 1 above)
Comment (amarantha-k) [reply modules/release-notes/pages/relnotes.adoc]: Good catch. Thanks!

**Review by arod1987 — APPROVED** [review]

**Review by mihirkamdarqe — APPROVED** [review]

**Review by mcpaddy — COMMENTED** [review]
With regards to Tools it looks good.

**Pull request merged**